tensorflow_federated/python/research/robust_aggregation/robust_aggregation_test.py

```diff
@@ -0,0 +1,233 @@
+# Lint as: python3
+# Copyright 2019, Krishna Pillutla and Sham M. Kakade and Zaid Harchaoui.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test Robust Aggregation with Numpy Implementation."""
+
+from __future__ import absolute_import
+from __future__ import division
+
+import collections
+import numpy as np
+from six.moves import range
+import tensorflow as tf
+
+import tensorflow_federated as tff
+from tensorflow_federated.python.learning.framework import optimizer_utils
+from tensorflow_federated.python.learning import model_utils
+from tensorflow_federated.python.common_libs import py_typecheck
+from tensorflow_federated.python.tensorflow_libs import tensor_utils
+
+from rfa import build_stateless_robust_aggregation
```

Comment:
```suggestion
from tensorflow_federated.python.research.robust_aggregation import rfa
```

TFF prefers to only import modules, rather than symbols.